Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -469,9 +469,29 @@ function Unlock-ConfigurationSections(){
     $manager.CommitChanges()
 }
 
+function Remove-IdentityViewsFolder {
```

Comment:
I understand that you created this to remove the Identity Views folder, but the code inside the function seems more general purpose to me. Maybe this could be named `Remove-Folder` or something to that effect?

Reply:
Slightly concerned about function name conflicts by going a route like that

Reply:
Good suggestion

Reply:
Could be something like Remove-IdentityFolder(?)

Reply:
I think that's a great idea.

Reply:
It removes files and folders, but we only care that the views folder is removed.